cg/meta/upload/scout/rnafusion_config_builder.py

```diff
@@ -38,23 +37,17 @@ def build_load_config(self) -> None:
         for db_sample in self.analysis_obj.family.links:
             self.load_config.samples.append(self.build_config_sample(case_sample=db_sample))
 
-    def include_case_files(self):
+    def include_case_files(self) -> None:
         """Include case level files for rnafusion case."""
         LOG.info("Including RNAFUSION specific case level files")
-        self.load_config.multiqc_rna = self.fetch_file_from_hk(self.case_tags.multiqc_report)
-        self.load_config.gene_fusion = self.fetch_file_from_hk(self.case_tags.genefusion_report)
-        self.load_config.gene_fusion_research = self.fetch_file_from_hk(
-            self.case_tags.genefusion_report_research
-        )
-        self.load_config.RNAfusion_inspector = self.fetch_file_from_hk(
-            self.case_tags.rnafusion_inspector
-        )
-        self.load_config.RNAfusion_inspector_research = self.fetch_file_from_hk(
-            self.case_tags.rnafusion_inspector_research
-        )
-        self.load_config.RNAfusion_report = self.fetch_file_from_hk(self.case_tags.rnafusion_report)
-        self.load_config.RNAfusion_report_research = self.fetch_file_from_hk(
-            self.case_tags.rnafusion_report_research
+        for scout_key in RNAFUSION_CASE_TAGS.keys():
+            self._include_file(scout_key)
+
+    def _include_file(self, scout_key) -> None:
+        """Include the file path associated to a scout configuration parameter if the corresponding housekeeper tags
+        are found. Otherwise return None."""
+        self.load_config.__setattr__(
+            scout_key, self.fetch_file_from_hk(self.case_tags.__getattribute__(scout_key))
```

Comment:
I think I might have led you down the wrong path with the dunder functions(`__func__`). I think this is the correct way:
```suggestion
        setattr(self.load_config, scout_key, self.fetch_file_from_hk(getattr(self.case_tags,scout_key))
```
Nevertheless, nice implementation 😁

Reply:
oh good. I didn't understand why using the dunder functions here. Much happier with this :)

Reply:
They can be used under very special circumstances, but generally avoid using them directly if possible